### PR TITLE
BZ1788218: RHEL workers update proc for y-stream

### PR DESCRIPTION
**Pull request by bobfuru**
https://bugzilla.redhat.com/show_bug.cgi?id=1788218
Adds wording to indicate that updates to an OCP minor version for RHEL worker nodes does not require any RPM packages to be included. It also follows the suggested (and already documented) path of running the OCP update first, which will update necessary packages to ensure that they are the correct versions for that version of OCP before updating RHEL worker nodes. I propose that that last part need not be documented but including the detail here for posterity (it came as a result of an internal Eng comment).

I also added the note about not upgrading to RHEL 8 because the statement was recently added elsewhere in the 4.7 release notes. If it's not pertinent here, please comment so I can remove it.

Lastly, I didn't attempt to update `compute` --> `worker` here because it is consistently referred to as the former throughout this section. If it should be updated, that would be grist for a separate PR.

Also updates `RHEL` --> `{op-system-base}` to use the defined attribute.

**Preview link (Updating RHEL compute machines in your cluster):** 
https://deploy-preview-35105--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-rhel-compute#rhel-compute-updating-minor_updating-cluster-rhel-compute

### CODE REVIEW

**Comment by bobfuru** [comment]
Hi @robertsand - PTAL

This is a first-pass attempt to address your BZ request. Do you find the placement and general wording to be helpful here? I couldn't determine any additional or alternate steps a user would need to take to accomplish the goal of performing a RHEL-based worker OS upgrade. Mostly, just the note that they should update OCP first and also use the RPM repo that uses the same OCP version as the cluster.

Please make any suggestions inline in the PR if that seems easiest for you. Thank you!

**Comment by gpei** [comment]
LGTM for the changes.

**Comment by bobfuru** [comment]
@rhatbob - This has been verified by QE. Does it look good to you?

**Comment by rhatbob** [comment]
> @rhatbob - This has been verified by QE. Does it look good to you?

Looks good to me!

File: modules/rhel-compute-updating.adoc [hunk 1]
```diff
@@ -2,26 +2,30 @@
...
-Because the RHEL machines require assets that are generated by the cluster to
-complete the update process, you must update the cluster before you update
-the RHEL compute machines in it.
+Because the {op-system-base} machines require assets that are generated by the cluster to complete the update process, you must update the cluster before you update the {op-system-base} worker machines in it.
 ====
 
-* You have access to the machine that you used to add the RHEL compute machines
-cluster. You must have access to the `hosts` Ansible inventory file that defines
-your RHEL machines and the `upgrade` playbook.
+* You have access to the machine that you used to add the {op-system-base} compute machines cluster. You must have access to the `hosts` Ansible inventory file that defines your {op-system-base} machines and the `upgrade` playbook.
```
Comment (codyhoag) [inline modules/rhel-compute-updating.adoc:26]: I was a little thrown off by the first sentence. I see this wording was there before this PR, but just wanted to flag it. Would something like this make more sense?

`You have access to the local machine that you used to add the {op-system-base} compute machines to your cluster.`

**Review by codyhoag — COMMENTED** [review]
Just one wording nit; all looks good otherwise!

File: modules/rhel-compute-updating.adoc (hunk 1 above)
Comment (bobfuru) [reply modules/rhel-compute-updating.adoc]: Thanks, @codyhoag - I agree that it was an awkward sentence and I like your suggested edit, so I've used your wording. 😁

**Pull request merged**